src/action_types/plugins.ts

```diff
@@ -5,4 +5,5 @@ import keyMirror from 'utils/key_mirror';
 export default keyMirror({
     RECEIVED_MARKETPLACE_PLUGINS: null,
     GET_MARKETPLACE_PLUGINS_FAILURE: null,
+    RECEIVED_PLUGIN_LOCATIONS: null,
```

Comment:
0/5 why are we sticking with "Plugin" terminology as opposed to Apps?

Reply:
Discussed offline. Will change Plugin references to App.